src/GraphQL.ApiTests/net50/GraphQL.MicrosoftDI.approved.txt

```diff
@@ -150,11 +150,21 @@ namespace GraphQL.MicrosoftDI
         public static GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> ResolveScoped<TSourceType, TReturnType>(this GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> builder, System.Func<GraphQL.IResolveFieldContext<TSourceType>, TReturnType?> resolver) { }
         public static GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> ResolveScopedAsync<TSourceType, TReturnType>(this GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> builder, System.Func<GraphQL.IResolveFieldContext<TSourceType>, System.Threading.Tasks.Task<TReturnType?>> resolver) { }
     }
+    public class ScopedFieldMiddleware : GraphQL.Instrumentation.IFieldMiddleware
+    {
+        public static readonly GraphQL.MicrosoftDI.ScopedFieldMiddleware Instance;
+        public ScopedFieldMiddleware(GraphQL.Instrumentation.IFieldMiddleware? innerMiddleware) { }
+        public System.Threading.Tasks.ValueTask<object?> ResolveAsync(GraphQL.IResolveFieldContext context, GraphQL.Instrumentation.FieldMiddlewareDelegate next) { }
+    }
```

Comment:
Now scoped fields just apply middleware to the field with this class.  It could be internal, however, it should replace ScopedFieldResolver so I left it public.